react-example/src/views/EmbeddedMsgs.tsx

```diff
@@ -141,98 +124,36 @@ export const EmbeddedMsgs: FC<Props> = () => {
       >
         {messages.length > 0 ? (
           messages.map((message: any, index: number) => {
-            const data = message?.elements;
-            const primaryButton = data?.buttons?.[0];
-            const secondaryButton = data?.buttons?.[1];
-            const defaultAction = data?.defaultAction;
+            const data = message;
 
             switch (selectedButtonIndex) {
               case 0:
                 return (
                   <Card
                     key={index.toString()}
-                    cardStyle={{ margin: 0 }}
-                    title={data?.title}
-                    text={data?.body}
-                    imgSrc={data?.mediaUrl}
-                    primaryBtnLabel={primaryButton?.title}
-                    secondaryBtnLabel={secondaryButton?.title}
-                    onClickPrimaryBtn={() =>
-                      handleOpenUrl(
-                        primaryButton?.action?.type,
-                        primaryButton?.action?.data
-                      )
-                    }
-                    onClickSecondaryBtn={() =>
-                      handleOpenUrl(
-                        secondaryButton?.action?.type,
-                        secondaryButton?.action?.data
-                      )
-                    }
-                    onClickView={() =>
-                      handleOpenUrl(defaultAction?.type, defaultAction?.data)
-                    }
+                    parentStyle={{ margin: 0 }}
+                    messageData={data}
                   />
                 );
 
               case 1:
                 return (
                   <Banner
                     key={index.toString()}
-                    BannerStyle={{ margin: 0 }}
-                    title={data?.title}
-                    text={data?.body}
-                    imgSrc={data?.mediaUrl}
+                    parentStyle={{ margin: 0 }}
```

Comment:
What is "parentStyle"? Just want to be sure I understand what this is / why it's named that way

Reply:
The 'parentStyle' will set the style to the parent view of all the components like background,  borderRadius, etc.